packages/twenty-server/src/engine/metadata-modules/utils/clean-object-metadata.util.ts

```diff
@@ -0,0 +1,10 @@
+import omit from 'lodash.omit';
+
+import { ObjectMetadataInterface } from 'src/engine/metadata-modules/field-metadata/interfaces/object-metadata.interface';
+
+import { ObjectMetadataItemWithFieldMaps } from 'src/engine/metadata-modules/types/object-metadata-item-with-field-maps';
+
+export const cleanObjectMetadata = (
```

Comment:
not a big fan of this util! title is not explicit. 
maybe we could add it to the relation-metadata module and name it: transformObjectMetadataForRelation?

Reply:
does not seem to be fixed!